### PR TITLE
ISSD-2247 [PRM] Oppty and Renewal listings are not showing correct data to CAMs

### PR DESCRIPTION
**Pull request by thaynam**

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-fields/19-opportunity-partner-role-sf.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 		{
 			"DBType": "Boolean",
 			"businessType": "Boolean",
-			"externalReferenceCode": "Active__c",
+			"externalReferenceCode": "Acitve_Partner__c",
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-fields/19-opportunity-partner-role-sf.json:6]:
```suggestion
			"externalReferenceCode": "Active_Partner__c",
```

Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-fields/19-opportunity-partner-role-sf.json:6]:
```suggestion
			"externalReferenceCode": "Active_Partner__c",
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-fields/19-opportunity-partner-role-sf.json (hunk 1 above)
Comment (thaynam) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-fields/19-opportunity-partner-role-sf.json:6]: I requested this change to the salesforce admins, waiting their confirmation.

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:test:sf

**Comment by jairmedeiros** [comment]
ci:forward:force

**Comment by jairmedeiros** [comment]
ci:forward:force

**Comment by jairmedeiros** [comment]
ci:forward:force

**Comment by jairmedeiros** [comment]
ci:forward:force

**Pull request closed without merging**